installer/centos/anaconda_installer.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# get install prefrix from cmd line
+if test "$#" -ne 1; then
+    echo "please specify an install prefix"
+fi
+
+# expand provided path
+if [[ ! "$1" =~ ^/ ]]
+    then
+        install_prefix="$PWD/$1"
+    else
+        install_prefix="$1"
+fi
+echo "Installing hnn, iv and nrn into $install_prefix"
+
+# check system dependencies.
+# TODO: What else should be checked for
+
+# check that mpicc is on the path
+if ! type -P mpicc &>/dev/null; then
+    echo "mpicc is not in\$PATH"
+    echo "Please install it and add to the PATH environment variable"
+fi
+
+
+# create conda environment
+# TODO: Replace with enviroment.yml file?
+conda create --name hnn python=3 cython scipy numpy matplotlib pyqt pyqtgraph pyopengl libtool=2.4.2
+source activate hnn
```

Comment:
I don't think it's a good idea to force a virtual environment. You can give an `environment.yml` file so users are free to create one if they want. But forcing the virtual environment means they have different versions of the same package on their system. 

So for instance, I could have a `numpy=1.13` in the mne virtual environment and `numpy=1.19` in the hnn virtual environment. Now, if I do some computations, I could get different results simply because I'm in different virtual environments.